store_app/views/homepage.py

```diff
@@ -0,0 +1,7 @@
+from app import app
+
+
+@app.route('/')
+@app.route('/home')
+def home():
```

Comment:
If you have time, you could use these routes to explain how to use this application.